content/foundation/glossary.md

```diff
@@ -414,6 +454,33 @@ given to their opinions grow.
 
 </dd>
 
+<dt>MIPMC  {#MIPMC}</dt>
```

Comment:
As mentioned in the comments I don't think we need new acronyms, "member of `<something>` is good enough

Reply:
I agree with @bdelacretaz